### ISSUE TITLE
DatePicker i18n is not working anymore

### ISSUE THREAD

**Issue by lolytxd** [vaadin/flow#4224]
Vaadin version: 10.0.0.rc1

### Description
None of elements in DatePickerI18n works. Month, weekdays, weekdays short names are in English, despite the fact that i set them. This applies also to words 'calendar', 'week', 'clear', 'today' and 'cancel'. Also first day of week is not changing.

### Actual outcome
![image](https://user-images.githubusercontent.com/11396254/40917849-eb4cfaf8-6804-11e8-8097-d48ef4821357.png)

### Steps to reproduce
For this example I used project from https://vaadin.com/start/v10-project-base

I changed MainView class like this:
```
@HtmlImport("styles/shared-styles.html")
@Route("")
public class MainView extends VerticalLayout {

	public MainView() {
		DatePicker datePickerFrom = datePicker("Date from");
		DatePicker datePickerTo = datePicker("Date to");
		add(datePickerFrom);
		add(datePickerTo);
	}

	private DatePicker datePicker(String caption) {
		DatePicker datePicker = new DatePicker(caption);
		datePicker.getElement().getStyle().set("--lumo-space-m", "0px");
		datePicker.setI18n(new DatePickerI18n().setWeek("tydzień")
				.setCalendar("kalendarz").setClear("wyczyść")
				.setToday("dziś").setCancel("anuluj").setFirstDayOfWeek(1)
				.setMonthNames(Arrays.asList("styczeń", "luty", "marzec",
						"kwiecień", "maj", "czerwiec", "lipiec", "sierpień",
						"wrzesień", "październik", "listopad", "grudzień"))
				.setWeekdays(Arrays.asList("poniedziałek", "wtorek", "środa",
						"czwartek", "piątek", "sobota", "niedziela"))
				.setWeekdaysShort(Arrays.asList("niedz", "pon", "wt", "śr", "czw", "pt",
						"son")));
		datePicker.setWidth("calc(50% - .25rem)");
		return datePicker;
	}
}
```

### Browsers Affected
- [x] Chrome
- [x] Firefox 60.0.1 x64
- [x] Edge 42.17134.1.0

**Comment by ZheSun88** [comment]
That should be fixed with https://github.com/vaadin/vaadin-date-picker-flow/pull/87 , which will be included in the next release (10.0.0.rc2)

**Comment by ZheSun88** [comment]
Issue moved to [vaadin/vaadin-date-picker-flow #89](https://github.com/vaadin/vaadin-date-picker-flow/issues/89) via [**ZenHub**](https://www.zenhub.com/)
